src/tinyformat.h

```diff
@@ -495,7 +495,11 @@ namespace detail {
 class FormatArg
 {
     public:
-        FormatArg() {}
+        FormatArg()
+             : m_value(NULL),
```

Comment:
Should this be `NULL` or `nullptr`? IIRC we prefer to use `nullptr` instead of `NULL` and tinyformat is something that we have made our own changes to before.

Reply:
In https://github.com/c42f/tinyformat/pull/43, it was originally `nullptr`, then changed to `NULL`. However if `nullptr` is preferred we'll use that, as we are already maintaining our own changes to this code.

Reply:
In the [Developer Notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md) is written "nullptr is preferred over NULL or (void*)0."

Reply:
Updated to use `nullptr`.